data-prepper-plugins/otel-trace-source/README.md

```diff
@@ -93,6 +93,16 @@ curl -k -H 'Content-Type: application/json; charset=utf-8'  -d '{"resourceSpans"
 ### Counter
 - `requestTimeouts`: measures total number of requests that time out.
 - `requestsReceived`: measures total number of requests received by otel trace source.
+- `successRequests`: measures total number of requests successfully processed by otel trace source plugin.
+- `badRequests`: measures total number of requests with invalid format processed by otel trace source plugin.
+- `requestsTooLarge`: measures total number of requests of which the number of spans in the content is larger than the buffer capacity.
```

Comment:
We should clarify that it is not just the buffer capacity but the available space left in the buffer. 
```
... is larger than the available buffer capacity.
```

Reply:
Actually this is for the corner case that request payload happens to be larger than the buffer capacity. If the payload is only larger than the current remaining capacity, request will be timed out after the wait.